components/form-switch/examples/30-responsive.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { FormSwitch } from '../src';
+
+export default () => (
+	<>
+		<h3>This breakpoint and wider</h3>
+		<FormSwitch name="ex1" id="ex1" size={['medium', 'xlarge']} block>
+			Extra large for MD breakpoint and wider
+		</FormSwitch>
+		<FormSwitch name="ex2" id="ex2" size={['small', 'small', 'xlarge']} block>
```

Comment:
Should we say "null" for consistency or leave it as "small" to show it works both ways?
size={['small', null, 'xlarge']}